src/luarocks/fs/lua.lua

```diff
@@ -11,16 +11,16 @@ local dir = require("luarocks.dir")
 local util = require("luarocks.util")
 local path = require("luarocks.path")
 
-local socket_ok, zip_ok, unzip_ok, lfs_ok, md5_ok, posix_ok, _
-local http, ftp, lrzip, luazip, lfs, md5, posix
+local socket_ok, zip_ok, unzip_ok, lfs_ok, digest_ok, posix_ok, _
+local http, ftp, lrzip, luazip, lfs, digest, posix
 
 if cfg.fs_use_modules then
    socket_ok, http = pcall(require, "socket.http")
    _, ftp = pcall(require, "socket.ftp")
    zip_ok, lrzip = pcall(require, "luarocks.tools.zip")
    unzip_ok, luazip = pcall(require, "zip"); _G.zip = nil
    lfs_ok, lfs = pcall(require, "lfs")
-   md5_ok, md5 = pcall(require, "md5")
+   digest_ok, digest = pcall(require, "digest")
    posix_ok, posix = pcall(require, "posix")
```

Comment:
If you care about compatibility with future Tarantool versions, where digest could be removed, then why no to check for md5 if digest is not ok?
```Lua
-- ...
if not digest_ok then
    md5_ok, md5 = pcall(require, "md5")
end
-- ...
```

Reply:
I've updated the patch to use both ways. Hope it ends the discussion